lib/reek/context/module_context.rb

```diff
@@ -20,6 +20,11 @@ def node_instance_methods
       def descriptively_commented?
         CodeComment.new(exp.leading_comment).descriptive?
       end
+
+      def namespace_module?
+        return unless (contents = exp.children.last)
+        !contents.find_nodes([:def, :defs], [:class, :module]).any?
+      end
```

Comment:
I’m not a huge fan of assignment-in-conditional(-in-parens-to-side-step-RuboCop), nor of `!…any?`. How about:

```Ruby
def namespace_module?
  contents = exp.children.last
  contents && contents.find_nodes([:def, :defs], [:class, :module]).empty?
end
```